great_expectations/expectations/row_conditions.py

```diff
@@ -10,7 +12,10 @@
     alphas,
 )
 
-from great_expectations.exceptions import GreatExpectationsError
+from great_expectations.exceptions import (
```

Comment:
@anthonyburdi Minor: my personal preference is to import all exceptions as `ge_exceptions`. :)

Reply:
Good catch - didn't mean to break the pattern.